contrib/kind-common

```diff
@@ -326,7 +326,7 @@ install_kubevirt() {
     # vX.Y.Z - install specific stable (i.e v1.3.1)
     # nightly - install newest nightly
     # nightly tag - install specific nightly (i.e 20240910)
-    KUBEVIRT_VERSION=${KUBEVIRT_VERSION:-"stable"}
+    KUBEVIRT_VERSION=${KUBEVIRT_VERSION:-"nightly"}
```

Comment:
Should we wait for a  kubevirt release ?

Reply:
https://github.com/kubevirt/sig-release/blob/main/releases/v1.4/schedule.md
lets wait, rc should be any day

Reply:
disregard, stable.txt wont have it, checking about alternatives (i.e new file on that path etc)

Reply:
yep